.travis.yml

```diff
@@ -8,6 +8,7 @@ go_import_path: goteleinfo
 
 env:
 - GLIDE_VERSION=v0.12.3
+- GLIDE_VERSION=v0.12.3 ARCH=arm GIMME_OS=linux GIMME_ARCH=arm
```

Comment:
I don't think `GIMME_*` variables are not needed here because Go has already ben installed